CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Bitcoind: more parallelism in requests, for very slow nodes.
 - Testing: fixed logging, cleaner interception of bitcoind, minor fixes.
 - JSON API: `invoice` now adds route hint to invoices for incoming capacity (RouteBoost), and warns if insufficient capacity.
+- CLI: `help` prettified, and falls back to internal help if manual page missing.
```

Comment:
oh it's not prettified yet, it's just spitting out that config-style output for now. I should update that commit description...

Reply:
I'm going to prettify the fallback help in a future PR since I need to refactor a few things first.